source/features/next-scheduled-github-action.tsx

```diff
@@ -65,7 +65,10 @@ async function init(): Promise<false | void> {
 
 		const nextTime = parseCron.nextDate(workflows[workflowName]);
 		if (nextTime) {
-			workflowListItem.append(<em>(next <relative-time datetime={nextTime.toString()}/>)</em>);
+			const relativeTime = <relative-time datetime={nextTime.toString()}/>;
+			workflowListItem.append(<em>(next {relativeTime})</em>);
+			workflowListItem.parentElement!.classList.add('tooltipped', 'tooltipped-e');
+			workflowListItem.parentElement!.setAttribute('aria-label', 'Next ' + relativeTime.textContent!.toString());
```

Comment:
It seems this isn't reliable:
![image](https://user-images.githubusercontent.com/46634000/109997386-6f964580-7d10-11eb-9abe-50510181c894.png)
Is there an easy way to wait for a custom element to have a text content? `MutationObserver` would be an overkill...

Reply:
This is due to `relative-time` not setting it right away, so the solution should be simple

<img width="415" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/110002230-c719be80-7cda-11eb-9bf8-13254433f947.png">